FF16Tools.Files/Timelines/Elements/Battle/TimelineElement_1083.cs

```diff
@@ -0,0 +1,39 @@
+﻿using FF16Tools.Files.Timelines.Chara;
+
+using Syroot.BinaryData;
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FF16Tools.Files.Timelines.Elements.Battle;
+
+public class TimelineElement_1083 : TimelineElementBase, ISerializableStruct
+{
+    public TimelineElement_1083()
+    {
+        UnionType = TimelineElementType.kTimelineElem_1083;
+    }
+
+    public int CommandId { get; set; }
+
+    public override void Read(SmartBinaryStream bs)
+    {
+        ReadMeta(bs);
+
+        CommandId = bs.ReadInt32();
+    }
```

Comment:
Consider also skipping the padding explicitly
Ideally Read/Write/GetSize should all match up.